src/libraries/shims/System.Net.ServicePoint/src/System.Net.ServicePoint.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>
```

Comment:
You must not change the public key token of the assembly. It was "Open" before. As open is already the default, just remove the property:

```suggestion
```